apps/client/src/app/core/event/mouse-wheel/mouse-wheel.directive.ts

```diff
@@ -0,0 +1,35 @@
+import { Directive, Output, HostListener, EventEmitter } from '@angular/core';
+
+@Directive({ selector: '[mouseWheel]' })
+export class MouseWheelDirective {
+  @Output() mouseWheelUp = new EventEmitter();
+  @Output() mouseWheelDown = new EventEmitter();
+
+  @HostListener('mousewheel', ['$event']) onMouseWheelChrome(event: any) {
+    this.mouseWheelFunc(event);
+  }
+
+  @HostListener('DOMMouseScroll', ['$event']) onMouseWheelFirefox(event: any) {
+    this.mouseWheelFunc(event);
+  }
+
+  @HostListener('onmousewheel', ['$event']) onMouseWheelIE(event: any) {
+    this.mouseWheelFunc(event);
+  }
+
+  mouseWheelFunc(eventUi: any) {
```

Comment:
```suggestion
  mouseWheelFunc(eventUi: any): void {
```

Even if we don't know the type of the event, we can type the return type as void, making it easier to read as we do'nt have to look for a return 🙂